src/Uniphar.Dotnet.Cli/Uniphar.Dotnet.Cli.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net8.0;net9.0</TargetFramework>
```

Comment:
Needs to be `<TargetFrameworks>` since you're multi-targeting.

Reply:
Mistake because of the hour... I didn't check if it is compiled or the CI action.